### PR TITLE
Added mangerController logic

### PR DESCRIPTION
**Pull request by alexkaroleva42**

### CODE REVIEW

File: controllers/managerController.js [hunk 1]
```diff
@@ -1,20 +1,80 @@
...
+        }
+
+        const candidate = await Territory.findOne({ where: { name } });
+
+        if (candidate) {
+            return next(ApiError.badRequest('Incorrect request: Territory with such name has already been created'))
+        }
+
+        const newTerritory = await Territory.create({
+            name
```
Comment (mastafit) [inline controllers/managerController.js:74]: You should catch an error in case it would occur in this part of the code

**Review by mastafit — CHANGES_REQUESTED** [review]
Fix this and commit with the changes

File: controllers/managerController.js (hunk 1 above)
Comment (alexkaroleva42) [reply controllers/managerController.js]: Ok

**Review by mastafit — APPROVED** [review]
Nice!

**Pull request merged**